### ISSUE TITLE
Timestamp fragment rendered as Emoji 

### ISSUE THREAD

**Issue by chrisross** [yoshuawuyts/vmd#101]
A fragment of this markdown markup (`:38:`) is interpreted as an Emoji.

```markdown
- Uses the classic rule builder (as at 2017-06-22 12:38:11)
```

Screenshot:
![image](https://user-images.githubusercontent.com/3296970/28401484-a2416c06-6d5d-11e7-951c-0f83739b3665.png)

Rendered HTML (copied from Chrome dev tools):
```html
<li>Uses the classic rule builder (as at 2017-06-22 12<img src="emoji://38" alt=":38:" title=":38:" align="absmiddle" class="emoji">11)</li>
```

Assuming all timestamps using the standard format are also impacted.

[Another tool](https://github.com/joeyespo/grip) using Github markdown has no issue

**Comment by josephfrazier** [comment]
I just encountered a similar problem with the following text:

```markdown
NOTE: here's a list of sources:
```

which renders as:

![vmd_bug](https://user-images.githubusercontent.com/6473925/29075327-01612b2e-7c20-11e7-8486-c92ebb7a01a3.png)

I imagine the underlying problem is in emojify.js, which appears to be unmaintained: https://github.com/emojione/emojify.js/tree/58e651c6e9b89cde35e6ccfa4f1e43374982789c#emojifyjs

**Comment by maxkueng** [comment]
It's fixed in vmd 1.29.2

Thanks for reporting the problem
